package-lock.json

```diff
@@ -10612,9 +10612,9 @@
       "dev": true
     },
     "node_modules/@tanstack/table-core": {
-      "version": "8.9.3",
-      "resolved": "https://registry.npmjs.org/@tanstack/table-core/-/table-core-8.9.3.tgz",
-      "integrity": "sha512-NpHZBoHTfqyJk0m/s/+CSuAiwtebhYK90mDuf5eylTvgViNOujiaOaxNDxJkQQAsVvHWZftUGAx1EfO1rkKtLg==",
+      "version": "8.10.3",
```

Comment:
@atmgrifter00 could you do whatever additional validation you think is necessary for the tanstack update? This could include
-  manual testing of the storybook build
-  looking at release notes for breakages or new features we want to use or bug fixes that allow us to remove workarounds

Note: this is just a package-lock update, meaning it affects the versions used by the repo's pipeline but it doesn't affect the versions that clients like SLE are required to install